### PR TITLE
Update locality and neighbourhood records in SA

### PR DESCRIPTION
**Pull request by stepps00**
Will fix https://github.com/whosonfirst-data/whosonfirst-data/issues/1909.

This PR will:

- [x] Add polygons to populous localities in Saudi Arabia
- [x] Add neighbourhood records to some major cities in Saudi Arabia
- [x] Update a handful of bunk locality/neighbourhood records in WOF
- [x] Add localizations to updated features
- [x] PIP records to update hierarchies
- [x] Validate records

### CODE REVIEW

**Comment by stepps00** [comment]
Aside from some localized names I'd like to add to this PR and PIP work, these changes are ready for review. In total, there will be 87 locality polygons and 878 neighbourhood polygons in this repo once this PR is merged.

Any locality with a population over 50k will have a polygon, and the following localities now have neighbourhood polygons:

- Al Qunfudhah
- Hafar al Batin
- Buraydah
- Ha'il

Locality polygons in SA (with labels):
Screen Shot 2022-08-01 at 9 12 40 AM

Neighbourhood polygons in SA:
Screen Shot 2022-08-01 at 9 13 09 AM

File: data/112/587/998/3/1125879983.geojson [hunk 1]
```diff
@@ -63,7 +67,7 @@
         }
     ],
     "wof:id":1125879983,
-    "wof:lastmodified":1566638482,
+    "wof:lastmodified":1659047515,
     "wof:name":"shokhaib\u064d",
```
Comment (nvkelso) [inline data/112/587/998/3/1125879983.geojson:71]: Nit: Titlecase names, and preferably ASCII7

File: data/112/592/443/9/1125924439.geojson [hunk 2]
```diff
@@ -84,7 +88,7 @@
         }
     ],
     "wof:id":1125924439,
-    "wof:lastmodified":1566638479,
+    "wof:lastmodified":1659047515,
     "wof:name":"\u0162urayf",
```
Comment (nvkelso) [inline data/112/592/443/9/1125924439.geojson:92]: Nit: lots of non-ASCII values in wof:name in this PR

**Review by nvkelso — APPROVED** [review]
Headed in the right direction. A few nits for wof:name ASCII values but polygons and rest LGTM

**Comment by stepps00** [comment]
The most recent commit includes name translations in Arabic. I'll review some of the ASCII values, PIP and validate this PR, and merge once that work is complete.

File: data/112/587/998/3/1125879983.geojson (hunk 1 above)
Comment (stepps00) [reply data/112/587/998/3/1125879983.geojson]: fixed in [e6a2f2d](https://github.com/whosonfirst-data/whosonfirst-data-admin-sa/pull/19/commits/e6a2f2d9a689c22d23c6ceabb5039d78bcfad61a)

File: data/112/592/443/9/1125924439.geojson (hunk 2 above)
Comment (stepps00) [reply data/112/592/443/9/1125924439.geojson]: fixed in [e6a2f2d](https://github.com/whosonfirst-data/whosonfirst-data-admin-sa/pull/19/commits/e6a2f2d9a689c22d23c6ceabb5039d78bcfad61a)

**Comment by stepps00** [comment]
The most recent commit includes all the PIP work / hierarchy updates needed for this branch. I've also validated all the changes in this PR.

Will review the changes again, then merge.

**Pull request merged**